docs/reference-manual.mdk

```diff
@@ -1521,6 +1557,60 @@ way to do it is by hand.
 [quantor]: http://fmv.jku.at/quantor/
 [ocaml-qbf]: https://github.com/c-cube/ocaml-qbf
 
+### Usage for Modal Logic mode {#usage-modal-logic}
+
+The language accepted by the Modal Logic mode is described in [#modal-logic-formulas].
+
+By default, `touist` is able to translate problems into the
+`InToHyLo` format with the `--modal-logic=SYSTEM` flag. In this mode,
+`SYSTEM` can be any non-whitespace string of characters. `SYSTEM` will simply be used
+for selecting the appropriate Modal Logic system and solver.
+
+It is necessary to have Linux OS in order to use Modal Logic S5 in `touist`.
```

Comment:
Why is Linux required? Isn't it possible to run it on macOS?

Reply:
I see... I never managed to find a way to download s5cheetah anyways, https://square16.org/tools/s5cheetah/ returns a 404 😁

<img width="1980" height="1312" alt="Capture d’écran 2025-07-19 à 21 56 01-fs8" src="https://github.com/user-attachments/assets/1be41808-301c-4e4a-bc1e-66966386abf1" />